builtin/mainmenu/content/dlg_package.lua

```diff
@@ -232,6 +233,57 @@ local function get_formspec(data)
 		else
 			table.insert_all(formspec, {"label[2,2;", fgettext("Loading..."), "]"} )
 		end
+	elseif current_tab == 4 then
+		if not package.releases and not data.releases_error and not data.releases_loading then
+			data.releases_loading = true
+
+			contentdb.get_package_releases(package, function(releases)
+				if not releases then
+					data.releases_error = true
+				end
+				ui.update()
+			end)
+		end
+
+		local function markup(text)
+			text = text:gsub("%*%*([^%s][^*]-[^%s])%*%*", "<b>%1</b>")
+			text = text:gsub("%*([^%s][^*]-[^%s])%*", "<i>%1</i>")
+			text = text:gsub("* ", "<b>•</b> ")
+			text = text:gsub("- ", "<b>•</b> ")
+			return text
+		end
+
+		local hypertext = {}
+		local insert = table.insert
+		if package.releases then
+			for _, def in pairs(package.releases) do
+				insert(hypertext,
+					"<b>"..core.hypertext_escape(def.title).."</b> ("..def.release_date:gsub("T.*", "")..")\n"..
+					fgettext("$1 downloads",
+						def.downloads)..".\n"
+				)
+
+				if def.release_notes then
+					insert(hypertext, core.hypertext_escape(
+							markup((def.release_notes
+								:gsub("<!--.-%-%->", "")
```

Comment:
just so you know (irrelevant because this code is obsolete anyways), this should have been

```suggestion
								:gsub("<!%-%-.-%-%->", "")
```